packages/embark/src/lib/modules/deployment/index.js

```diff
@@ -104,10 +104,13 @@ class DeployManager {
                 return done();
```

Comment:
This and line 100 need to change to `next` too

Reply:
Both works fine as an early `done()` kind of acts like an early `return`. Will update.